packages/zip-it-and-ship-it/src/runtimes/node/in_source_config/properties/region.ts

```diff
@@ -0,0 +1,17 @@
+export const REGIONS = [
+  'ap-northeast-1',
+  'ap-southeast-1',
+  'ap-southeast-2',
+  'ca-central-1',
+  'eu-central-1',
+  'eu-north-1',
+  'eu-south-1',
+  'eu-west-1',
```

Comment:
does it make sense to include the regions that we don't support in self-serve mode ([docs](https://docs.netlify.com/functions/optional-configuration/?fn-language=ts#region))? The reason we wanted folks to go through Support is to assess whether customers _really_ need these

Reply:
Ah, good catch. I copied this list from the UI but didn't realise two of the regions had a boolean set to `false`. I've removed them in https://github.com/netlify/build/pull/5901/commits/8546f9cab4b64bd1ed7ed73c2bba8b5b9f31790f.

Reply:
Oops, there's a third one. Fixed in https://github.com/netlify/build/pull/5901/commits/30be1303fbb8498fc60a3192fb6adc6ddc615e2b.